providers/shared/components/objectsql/id.ftl

```diff
@@ -1,5 +1,10 @@
 [#ftl]
 
+[@addComponentDeployment
```

Comment:
Should everything have a priority for consistency?

Reply:
Since its the default priority instead of the actual priority I thought it was better to leave it. The default is sent in the macro and would be available in the component schema

Reply:
ok - that works